docs/deployment/aws/ecs.md

```diff
@@ -16,7 +16,7 @@ The example handles tasks such as generating a docker image and setting up an ap
     $ git clone https://github.com/developmentseed/titiler.git
 
     # install cdk dependencies
-    $ cd titiler/deployment/aws && pip install -r requirements.txt
+    $ cd titiler/deployment/AWS && pip install -r requirements.txt
```

Comment:
Why this change? Isn't the directory lower case `aws`?

Reply:
Ah I was on mobile and the app showed lower case `aws` in the file name. I'd strongly suggest making the directory lower case instead.

Reply:
To expand a little more: I think Windows and Mac filesystems are case-insensitive, and Linux is case-sensitive. So in a previous project with users on multiple OSes, we found hard-to-debug issues due to case sensitivity. So I'd argue it's best practice to just always use lower case.

Reply:
made is lower case https://github.com/developmentseed/titiler/commit/c952bd445e545bc8390c413d5f6fb0b6258dd071

Reply:
😬 in fact I didn't fixed this yet!

Reply:
done https://github.com/developmentseed/titiler/commit/553ac43c4e026b70e26a189520db74f90c211c06 cc @fredliporace